### PR TITLE
Update MPC Core Kit v2.3.0

### PR DESCRIPTION
**Pull request by shahbaz17**

### CODE REVIEW

File: docs/sdk/core-kit/mpc-core-kit/usage.mdx [hunk 1]
```diff
@@ -10,11 +10,12 @@ provider given by Web3Auth to sign transactions and interact with the blockchain
 
 Natively, the instance of `Web3AuthMPCCoreKit` (referred to as `coreKitInstance` in our examples) returns the following functions:
 
-Authentication Functions:
+Authentication and Handling Functions:
 
-- `loginWithOAuth()` - Logs in the user in OAuth Intrisic Flow.
-- `loginWithJWT()` - Logs in the user with a JWT based ID Token.
+- `loginWithOAuth()` - Logs in the user in OAuth Intrinsic Flow.
```
Comment (himanshuchawla009) [inline docs/sdk/core-kit/mpc-core-kit/usage.mdx:15]: intrinsic or implicit?

File: docs/sdk/core-kit/mpc-core-kit/usage.mdx [hunk 2]
```diff
@@ -51,6 +54,41 @@ Additional Helper Functions:
...
+
+### Parameters
+
+```js
+public getPublicSync: () => Buffer = () => {
+  let { tssPubKey } = this.state;
+  if (tssPubKey.length === FIELD_ELEMENT_HEX_LEN + 1) {
+    tssPubKey = tssPubKey.subarray(1);
+  }
+  return Buffer.from(tssPubKey);
```
Comment (himanshuchawla009) [inline docs/sdk/core-kit/mpc-core-kit/usage.mdx:82]: why we are defining sdk code of the function in docs, any reason?

File: docs/sdk/core-kit/mpc-core-kit/usage.mdx [hunk 3]
```diff
@@ -51,6 +54,41 @@ Additional Helper Functions:
...
+};
+```
+
+#### Usage
+
+```ts
+await coreKitInstance.getPublicSync();
```
Comment (himanshuchawla009) [inline docs/sdk/core-kit/mpc-core-kit/usage.mdx:89]: @ieow is this function returns final public key of the user?

File: docs/sdk/core-kit/mpc-core-kit/usage.mdx [hunk 4]
```diff
@@ -51,6 +54,41 @@ Additional Helper Functions:
...
+#### Usage
+
+```tsx
+await coreKitInstance.setTssWalletIndex(index);
+```
+
+## Get Public Sync
+
+#### `getPublicSync()`
+
```
Comment (himanshuchawla009) [inline docs/sdk/core-kit/mpc-core-kit/usage.mdx:73]: what is use of this function?

File: docs/sdk/core-kit/mpc-core-kit/usage.mdx [hunk 5]
```diff
@@ -51,6 +54,41 @@ Additional Helper Functions:
   - `mnemonicToKey()` - Converts a Mnemonic to a BN.
   - `keyToMnemonic()` - Converts a hex string to a Mnemonic.
 
+## Set TSS Wallet Index
+
+#### `setTssWalletIndex(accountIndex: number): void;`
+
+Sets the TSS Wallet Index for the User's account. This is a new feature that supports key derivation, allowing users to derive multiple private keys
```
Comment (himanshuchawla009) [inline docs/sdk/core-kit/mpc-core-kit/usage.mdx:61]: In mpc there are no Private keys, i think better description would be : "This function can be used to deterministically derive different wallet addresses by using account indexes. ". Using this function you can add multi address account support to your application.

Note: As an application, you need to persist this account index in application state. 

This feature is still in pre-release , so there can be some breaking changes in upcoming versions.

**Review by himanshuchawla009 — CHANGES_REQUESTED** [review]

File: docs/sdk/core-kit/mpc-core-kit/usage.mdx (hunk 3 above)
Comment (himanshuchawla009) [reply docs/sdk/core-kit/mpc-core-kit/usage.mdx:89]: also the public key will depends on account index being set, so for different account indexes this function will return different public keys. We should mention it in docs.

**Review by himanshuchawla009 — CHANGES_REQUESTED** [review]

File: docs/sdk/core-kit/mpc-core-kit/usage.mdx (hunk 1 above)
Comment (shahbaz17) [reply docs/sdk/core-kit/mpc-core-kit/usage.mdx]: Intrinsic.

It was a typo from earlier. I didn't check. Thanks for bringing this to my attention. Updated at all places.

File: docs/sdk/core-kit/mpc-core-kit/usage.mdx (hunk 4 above)
Comment (shahbaz17) [reply docs/sdk/core-kit/mpc-core-kit/usage.mdx:79]: Please check once again as per the new description below.

File: docs/sdk/core-kit/mpc-core-kit/usage.mdx (hunk 3 above)
Comment (shahbaz17) [reply docs/sdk/core-kit/mpc-core-kit/usage.mdx:92]: Updated!

File: docs/sdk/core-kit/mpc-core-kit/usage.mdx (hunk 5 above)
Comment (shahbaz17) [reply docs/sdk/core-kit/mpc-core-kit/usage.mdx]: Updated!

**Pull request merged**